### PR TITLE
Create ios_17.md

### PR DESCRIPTION
**Pull request by bkasman95**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+nav_title: iOS 17 Upgrade Guide
+article_title: iOS 17 Upgrade Guide
+page_order: 7
+platform: 
+  - iOS
+description: "This refernce article covers iOS 17, SDK updates, and more."
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:7]:
```suggestion
description: "This reference article covers iOS 17, SDK updates, and more."
```

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description: "This reference article covers iOS 17, SDK updates, and more."
+---
+
+## Changes in iOS 17
+
+### Link Tracking & UTM Parameter Stripping
+
+One of the important changes introduced at WWDC this year was the blocking UTM parameters in Safari. UTM parameters are pieces of code that are added to URLs, which are frequently used in marketing campaigns to measure the effectiveness of Email, SMS and other campaigns. 
+
+Since Braze encodes all links that utilize click-tracking, we do not foresee this impacting integrations.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:16]:
```suggestion
Braze email and SMS products encode links in a way that is not  impacted by this change.  
```

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Link Tracking & UTM Parameter Stripping
+
+One of the important changes introduced at WWDC this year was the blocking UTM parameters in Safari. UTM parameters are pieces of code that are added to URLs, which are frequently used in marketing campaigns to measure the effectiveness of Email, SMS and other campaigns. 
+
+Since Braze encodes all links that utilize click-tracking, we do not foresee this impacting integrations.  
+
+### Continued Privacy Changes
+
+Additionally, Apple announced its commitment to further expand the scope of App Tracking Transparency (ATT), which enables users to control whether an app can access their activity across various apps and websites belonging to other companies. With this in mind, Apple announced 2 new key features to support this.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:20]:
```suggestion
Apple announced its commitment to further expand the scope of App Tracking Transparency (ATT), which enables users to control whether an app can access their activity across various apps and websites belonging to other companies. With this in mind, Apple announced two new key features to support this.
```

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The first is the introduction of privacy manifests, which allows developers to outline the privacy practices of the third-party code (including third party SDKs) in their app in a standardized format. With privacy manifests, Apple also plans to offer additional privacy protection for users, by identifying applications referencing APIs that might be used for fingerprinting. If applications are using any one of these APIs, they will be required to list a “required reason” for using that API in the privacy manifest as well.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:22]: would be nice to add a link to apple's doc on privacy manifests

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:22]:
```suggestion
The first is the introduction of privacy manifests, which allows developers to outline the privacy practices of the their app - including third party SDKs - in a standardized format. With privacy manifests, Apple also plans to offer additional privacy protection for users, by identifying and blocking iOS APIs that might be used for fingerprinting. If applications are using any one of these APIs, they will be required to list a “required reason” for using that API in the privacy manifest as well. 
```

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+The second feature introduced is code signing, which allows developers using a third party SDK in their application to validate within XCode that it was signed by the same developer as previous versions. 
+
+Related to privacy in gerenal, Apple has also announced that they will release a list of third party SDKs that are considered “privacy impacting” later this year - these SDKs are expected to be considered having an especially high impact on user privacy by Apple.
+
+Unlike traditional tracking SDKs that are designed to monitor users across multiple websites and applications, the Braze SDK solely focuses on storing first-party data and strictly refrains from sharing to different companies.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/swift/ios_17.md:28]:
```suggestion
Unlike traditional tracking SDKs that are designed to monitor users across multiple websites and applications, the Braze SDK focuses on first-party data messaging and user experiences.

While we do not expect to be included in this list, we will monitor this closely and release any required updates if necessary.
```

File: _docs/_developer_guide/platform_integration_guides/swift/ios_17.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+nav_title: iOS 17 Upgrade Guide
+article_title: iOS 17 Upgrade Guide
+page_order: 7
+platform: 
+  - iOS
+description: "This reference article covers iOS 17, SDK updates, and more."
+---
+
+## Changes in iOS 17
```
[the remark this answers is not in the document]
Comment (bkasman95) [reply _docs/_developer_guide/platform_integration_guides/swift/ios_17.md]: Look ok now?

**Review by josh-mccrowell-braze — APPROVED** [review]

**Pull request merged**